### PR TITLE
refactor: replace dt_t* with DtBuilder (part 4)

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

**Comment by WalterBright** [comment]
Hah, finally saw what I did wrong. Why doing these in smaller chunks is worthwhile.

File: src/glue.c [hunk 1]
```diff
@@ -395,7 +401,11 @@ void genObjFile(Module *m, bool multiobj)
         bcov->Stype->Tcount++;
         bcov->Sclass = SCstatic;
         bcov->Sfl = FLdata;
-        dtnbytes(&bcov->Sdt, (m->numlines + 32) / 32 * sizeof(*m->covb), (char *)m->covb);
+
+        DtBuilder dtb;
+        dtb.nbytes((m->numlines + 32) / 32 * sizeof(*m->covb), (char *)m->covb);
```
Comment (andralex) [inline src/glue.c]: Would `+ 31` be enough instead of `+ 32`?

**Comment by andralex** [comment]
Auto-merge toggled on

File: src/glue.c (hunk 1 above)
Comment (WalterBright) [inline src/glue.c]: I think you're right. But such changes should be in a different PR.
